modules/openy_features/openy_location/modules/openy_loc_camp/src/CampHelper.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\openy_loc_camp;
+
+use Drupal\node\Entity\Node;
+use Drupal\node\NodeInterface;
+
+/**
+ * Class Camp Helper functions.
+ */
+class CampHelper {
+
+  /**
+   * Helper that returns the Camp CT associated to the landing page provided.
+   *
+   * @param \Drupal\openy_prgf_camp_menu\NodeInterface $node
+   *   The node.
+   */
+  public static function getLandingPageCampNode(NodeInterface $node) {
+    $camp = NULL;
+    if ($node->bundle() != 'landing_page') {
```

Comment:
this looks pretty weird, if we are talking about universal code.
At like 25 i see check for field exists. Could be safe to remove check for CT and rely on field availability only?